version1/02 - Object types.md

```diff
@@ -757,6 +759,30 @@ The following notation is used in the tables below:
+|           |           |                        |          |             |                  |  |      |      |       |      |      |        |                                                                                                                                |
+| M001      | string    | uuid                   |    x     |      x      |                  |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M600      | timestamp | opprettetDato          |    x     |      x      |                  |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M601      | string    | opprettetAv            |    x     |      x      |                  |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M601      | string    | opprettetAvBrukerIdent |    x     |      x      |                  |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M020      | string    | title                  |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M021      | string    | beskrivelse            |          |             |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M111      | date      | presedensDato          |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M628      | timestamp | presedensGodkjentDato  |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M629      | string    | presedensGodkjentAv    |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
```

Comment:
`presedensGodkjentAvIdent` is missing (if naming stays like this).

Reply:
Actually, seems like we would probably name the field presedensGodkjentAvBrukerIdent.

Reply:
Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6